### PR TITLE
Set the JDK 8 release version to avoid the ALPN issue

### PR DESCRIPTION
**Pull request by cescoffier**
With Java 8, set the release version to `jdk8u242-b08` to avoid the ALPN provider issue.

### CODE REVIEW

**Review by geoand — APPROVED** [review]

File: .github/workflows/ci-actions.yml [hunk 1]
```diff
@@ -65,6 +65,7 @@ jobs:
         uses: joschi/setup-jdk@b9cc6eabf7e7e3889766b5cee486f874c9e1bd2d
         with:
           java-version: 11
+          release: jdk8u242-b08
```
Comment (geoand) [inline .github/workflows/ci-actions.yml:68]: Do we need this?

Comment (Ladicek) [reply .github/workflows/ci-actions.yml:68]: ... in a step that installs Java __11__? :-)

**Comment by geoand** [comment]
JDK 11 failure on Windows has nothing to do with this PR:

```
2020-04-18T11:32:07.5369974Z [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 19.921 s <<< FAILURE! - in io.quarkus.it.amqp.AmqpConnectorTest
2020-04-18T11:32:07.5434605Z [ERROR] test  Time elapsed: 10.626 s  <<< ERROR!
```

**Comment by gsmet** [comment]
@cescoffier can we document here what's the issue? We need a version equal and newer or it's a new version that has the issue?

I would also add a comment where the hack is explaining why we are doing that.

Thanks.

**Comment by cescoffier** [comment]
@gsmet The CI started using OpenJDK 1.8.0-252 which contains some backport around ALPN (code coming from Java 9). Unfortunately, this new code breaks the ALPN detection, which means that Quarkus HTTP/2 features are, for now, only available on Java 1.8.0-242- and Java 11.  (another reason to drop Java 8 ?)

The ideal fix would be in Netty to improve the detection and avoid loading Jetty ALPN provider (which may not be in the classpath). This would need to be integrated in Vert.x and then in Quarkus.

**Pull request merged**